Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -15460,9 +15464,11 @@
 									"pm.environment.set(\"known:beginDate\",beginDate);\r",
 									"pm.environment.set(\"known:programName\", \"Bilingual\");\r",
 									"pm.environment.set(\"known:programTypeDescriptor\", \"uri://ed-fi.org/ProgramTypeDescriptor#Bilingual\");\r",
+									"pm.environment.set(\"known:programTypeDescriptor:encoded\", encodeURIComponent(\"uri://ed-fi.org/ProgramTypeDescriptor#Bilingual\"));\r",
```

Comment:
This part of the key was not included in the criteria originally, probably due to challenges with dealing with test with a descriptor URI on the query string. This approach encodes the string for use as a query string parameter.